net-libs/libwebsockets/metadata.xml

```diff
@@ -20,14 +20,20 @@
 	<use>
 		<flag name="access-log">Support generating Apache-compatible access logs</flag>
 		<flag name="cgi">Include CGI (spawn process with network-connected stdin/out/err) APIs</flag>
+		<flag name="dir">Enable directory scanning api support</flag>
+		<flag name="file-ops">Support file operations vfs</flag>
 		<flag name="client">The client part of the library and libwebsockets-test-client</flag>
+		<flag name="extensions">Compile with extensions</flag>
 		<flag name="generic-sessions">With the Generic Sessions plugin</flag>
 		<flag name="http2">Support the HTTP/2 protocol</flag>
 		<flag name="http-proxy">Support for rewriting HTTP proxying</flag>
+		<flag name="http-basic-auth">Support Basic Auth</flag>
 		<flag name="lejp">With the Lightweight JSON Parser</flag>
 		<flag name="libev">Support event loops via dev-libs/libev</flag>
 		<flag name="libevent">Support event loops via dev-libs/libevent</flag>
 		<flag name="libuv">Support asynchronous I/O via dev-libs/libuv</flag>
+		<flag name="mbedtls">Use mbedTLS replacement for OpenSSL</flag>
+		<flag name="testapps">Build the libwebsocket-test-apps</flag>
```

Comment:
Would it be possible to have USE=test and actually use these in src_test, or is that not feasible?

Reply:
I've investigated that before, even engaged the author on this, an it's not feasible.

Reply:
Worth a go. I remember us having that discussion actually. :(